### ISSUE TITLE
NameError: name 'os' is not defined

### ISSUE THREAD

**Issue by nlapenko** [TheLastBen/fast-stable-diffusion#1669]
Yesterday, I started the server after performing a fresh installation and setting everything up. I used it for an evening and closed it. However, this morning, I attempted to open the collab link but was unable to do so. I restarted the collab and selected "Start Stable-Diffusion," but encountered the following error: "NameError: name 'os' is not defined."

I believe this error may be due to a missing dependency or an issue with the code itself. I would appreciate any assistance you can provide in resolving this issue so that I can continue using the code.
<img width="1163" alt="Снимок экрана 2023-03-03 в 01 34 54" src="https://user-images.githubusercontent.com/78685488/222575337-167be4ed-8558-4e9f-83b2-fe26f8be8e64.png">

**Comment by nlapenko** [comment]
`---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
[<ipython-input-2-2ef9acacded3>](https://localhost:8080/#) in <module>
     16 
     17 with capture.capture_output() as cap:
---> 18   if not os.path.exists('/tools/node/bin/lt'):
     19     get_ipython().system('npm install -g localtunnel')
     20 

NameError: name 'os' is not defined`

**Comment by Rb-diff** [comment]
Ok, it isn't apparent if you did this from what you wrote. That looks like the error you get when any of the above cells in colab has not been run. when you restart a colab  after it has disconnected, you need to run every cell again. it's a fresh startup, you cannot just pause it. 

I'm sorry if this was not what you were asking and it sounded patronizing but whenever I get that error, that is the only way.

**Comment by nlapenko** [comment]
In other words, every time I need to use the stable diffusion interface, I need to restart all cells in the "fast_stable_diffusion_AUTOMATIC1111.ipynb" file. , not just the last one?

**Comment by TheLastBen** [comment]
@nlapenko Yes, because everytime you disconnect, the session is reset, so you need to bring back the session by running all the calls in order.

**Comment by nlapenko** [comment]
Hi, I have a problem again. Sorry, I'm unlucky. After disconnecting the collab, started a new session from the first item. Everything was fine. But now I got an error on the last item:
`No checkpoints found. When searching for checkpoints, looked at:
 - file /content/gdrive/MyDrive/sd/stable-diffusion-webui/model.ckpt
 - directory /content/gdrive/MyDrive/sd/stable-diffusion-webui/models/Stable-diffusion
Can't run without a checkpoint. Find and place a .ckpt or .safetensors file into any of those locations. The program will exit.
`

**Comment by Rb-diff** [comment]
check the box redownload original model?
<img width="487" alt="Screenshot 2023-03-03 163421" src="https://user-images.githubusercontent.com/125707228/222761713-f967bedd-8f25-4b96-b57f-7d1be7e228be.png">

**Comment by nlapenko** [comment]
I'm uploading the model via google disk with a link to it. Strangely enough, the problem solved itself after I restarted the last item 10 times in a row. Sorry for the inconvenience! Thanks.

**Comment by Rb-diff** [comment]
oh. here's a tip then. if you make any changes in your google drive. wait half a minute for it to take effecct and then press the little reload icon in the sidebar of the collab file structure. Google can be incredibly sluggish at updating and siometimes needs a little convincing.

<img width="290" alt="Screenshot 2023-03-03 164350" src="https://user-images.githubusercontent.com/125707228/222763827-93937457-cd6b-429a-8fd7-93f7b7389d73.png">
